cypress/e2e/CMS/$CreateAnImage/index.js

```diff
@@ -0,0 +1,172 @@
+    });
+
+And('user clicks on {string} button to select media', (media) => {
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find(`input#edit-submit[value="${media}"]`).click({ force: true })
+})
+
+And('user selects {string} from Contents dropdown', (selectOption) => {
+    cy.get(`.placeholder:contains("Contents")`).parent().find(`input[value="${selectOption}"]`).click({ force: true });
+    
+})
+
+And('user selects {string} from {string} dropdown {string} section', (dropDown, option, section) => {
+    cy.get(`div:contains("${section}")`).parent().parent().find(`.placeholder:contains("${option}")`).parent().find(`input[value="${dropDown}"]`).eq(0).click({ force: true })
+})
+
+Then('lead image is displayed with source {string}', (image) => {
+    cy.get('.centered-element img').should('have.attr','src').then((attrSrc) => {
+        expect(attrSrc.replace('.svg', '')).to.include(image.replace('.svg', ''));
+    })
+});
+
+And('caption reads {string}', (caption) => {
+    cy.get('.caption-container p').should('include.text',caption);
+});
+
+And('credit reads {string}', (credit) => {
+    cy.get('.image-photo-credit').should('include.text',credit);
+});
+
+And('{string} button is displayed', (enlarge) => {
+    cy.get('.article-image-enlarge').should('be.visible').should('have.text', enlarge);
+});
+
+Then('feature card image is displaying {string}', (image) => {
+    cy.get('.feature-primary.flex-columns.row img').should('have.attr','src').then((attrSrc) => {
+        expect(attrSrc.replace('.svg', '')).to.include(image.replace('.svg', ''));
+    })
+});
+
+Then('{int} item in the list has an image {string}', (item,image) => {
+    cy.get('.image.item-image > img').should('have.attr','src').then((attrSrc) => {
+        expect(attrSrc.replace('.svg', '')).to.include(image.replace('.svg', ''));
+    })
+});
+
+Then('the {string} appear as override', (override) => {
+     cy.get('div.multimedia-slot img').first().should('have.attr','src').then((attrSrc) => {
+        expect(attrSrc.replace('.svg', '')).to.include(override.replace('.svg', ''));
+     
+    })
+});
+
+And('user deletes {string} image', (image) => {
+    cy.get(`form[id^="views-form-media-media-page-list"]`).then(($content) => {
+        if ($content.find(`a:contains("${image}")`)) {
+            cy.get(`a:contains("${image}")`).parent().parent().find('input.form-checkbox').check();
+            cy.get(`input[value='Apply to selected items']`).first().click();
+            cy.get('h1:contains("Are you sure you want to delete this media item?")').should('be.visible');
+            cy.get(`input[value='Delete']`).click();
+            cy.get("div[role='contentinfo']").should('include.text', 'Deleted 1 item.');
+        }
+    });
+});
+
+
+And('the image {string} does not exist in the list of content', (image) => {
+    cy.get(`form[id^="views-form-media-media-page-list"]`).then(($content) => {
+            cy.get(`a:contains("${image}")`).should('not.exist');
```

Comment:
$content is not used - why do you need to go into jquery by chaining with then if you dont actually use jquery ?

Reply:
fixed